modules/juce_audio_plugin_client/LV2/juce_LV2_Wrapper.cpp

```diff
@@ -898,9 +898,9 @@ class JuceLv2Wrapper : public AudioPlayHead
 
         portAudioIns.insertMultiple (0, nullptr, numInChans);
         portAudioOuts.insertMultiple (0, nullptr, numOutChans);
-        portControls.insertMultiple (0, nullptr, filter->getNumParameters());
+        portControls.insertMultiple (0, nullptr, filter->getParameters().size());
 
-        for (int i=0; i < filter->getNumParameters(); ++i)
+        for (int i=0; i < filter->getParameters().size(); ++i)
```

Comment:
What purpose do these changes (and other `getNumParameters()` changes, etc.) serve? Just curious.

Reply:
The `getNumParameters()` function has been [deprecated](https://github.com/juce-framework/JUCE/blob/fbe95b0b073ccb589e28255cb0cd56e1382dc2c9/modules/juce_audio_processors/processors/juce_AudioProcessor.h#L1386), and I just got tired of having to scroll through a bunch of `-Wdeprecated-declarations` warnings to find the actual errors in my builds.

Reply:
That makes sense, I noticed that too and it was annoying. I'm not sure though if @falkTX would prefer to make these changes in a separate pull request or if doing it in this PR is fine. I know he maintains a [set of patches](https://github.com/DISTRHO/DISTRHO-Ports/tree/master/libs/juce-current/patches), so I'm thinking it might be beneficial to keep the changeset small so that it can be cleanly applied from a patch file.